packages/web-components/src/components/ic-text-field/ic-text-field.tsx

```diff
@@ -59,113 +59,113 @@ export class TextField {
+   * If `true`, the input will require a value.
    */
   @Prop() required: boolean = false;
   /**
-   * Specify whether the input is disabled.
+   * If `true`, the disabled state will be set.
    */
   @Prop() disabled: boolean = false;
 
   /**
-   * Specify whether the input is read only.
+   * If `true`, the read only state will be set.
    */
   @Prop() readonly: boolean = false;
 
   /**
-   * Provide a placeholder value to display.
+   * The placeholder value to be displayed.
    */
   @Prop() placeholder: string = "";
 
   /**
    * Specify whether the component fills the full width of the container.
-   * If true, this overrides the --input-width css prop.
+   * If `true`, this overrides the --input-width css prop.
    */
   @Prop() fullWidth: boolean = false;
 
   /**
-   * Hides the label and applies the required label value as an aria-label.
+   * If `true`, the label will be hidden and the required label value will be applied as an aria-label.
    */
   @Prop() hideLabel: boolean = false;
 
   /**
-   * Provide helper text to display additional field guidance.
+   * The helper text that will be displayed for additional field guidance.
    */
   @Prop() helperText: string = "";
 
   /**
-   * Specify the number of rows to transform the text field into a text area with a specific height.
+   * The number of rows to transform the text field into a text area with a specific height.
    */
   @Prop() rows: number = 1;
 
   /**
-   * Allow resizing of a multiline text area
+   * If `true`, the multiline text area will be resizeable.
    */
   @Prop() resize: boolean = false;
 
   /**
-   * Specify whether small styling is to be applied to the element.
+   * If `true`, the small styling will be applied to the element.
    */
   @Prop() small: boolean = false;
 
   /**
-   * Value of TextField.
+   * The value of TextField.
    */
   @Prop({ reflect: true, mutable: true }) value: string = "";
 
   /**
-   * Maximum number of characters that can be entered in the field.
+   * The maximum number of characters that can be entered in the field.
    */
   @Prop() maxLength: number = 0;
 
   /**
-   * Provide validation state - 'error' | 'warning' | 'success'
+   * The validation state - e.g. 'error' | 'warning' | 'success'.
    */
   @Prop() validationStatus: IcInformationStatusOrEmpty = "";
 
   /**
-   * Provide validation state - 'error' | 'warning' | 'success'
+   * The validation state - e.g. 'error' | 'warning' | 'success'.
    */
   @Prop() validationText: string = "";
 
   /**
-   * Show icon in input control - only applies when validationStatus ='success'
+   * If `true`, the icon in input control will be displayed - only applies when validationStatus ='success'.
    */
   @Prop() validationInline: boolean = false;
 
   /**
-   * Indicates whether and how the text value should be automatically capitalized as it is entered/edited by the user.
+   * The automatic capitalization of the text value as it is entered/edited by the user.
```

Comment:
Same as my other comment on the search bar
```suggestion
   * The automatic capitalisation of the text value as it is entered/edited by the user.
```